vimeo-networking/src/main/java/com/vimeo/networking/model/error/VimeoError.java

```diff
@@ -204,12 +204,12 @@ public ErrorCode getInvalidParameterErrorCode() {
         return getInvalidParameter() != null ? getInvalidParameter().getErrorCode() : null;
     }
 
-    public Exception getException() {
-        return mException;
+    public Throwable getThrowable() {
+        return mThrowable;
     }
 
-    public void setException(Exception exception) {
-        this.mException = exception;
+    public void setThrowable(Throwable throwable) {
```

Comment:
The setters and getters for exception are used in multiple places in the app. Is there a corresponding PR to update the app?

Reply:
I created a ticket for it, and I will tag you on it.